app/src/main/index.js

```diff
@@ -95,53 +115,90 @@ ipcMain.on('open-cropper-window', (event, size) => {
   } else {
     const {width = size.width, height = size.height} = settings.getSync('cropperWindow.size');
     const {x, y} = settings.getSync('cropperWindow.position');
-    cropperWindow = new BrowserWindow({
-      width: width + cropperWindowBuffer,
-      height: height + cropperWindowBuffer,
-      frame: false,
-      transparent: true,
-      resizable: true,
-      shadow: false,
-      x,
-      y
-    });
-    cropperWindow.loadURL(`file://${__dirname}/../renderer/views/cropper.html`);
-    cropperWindow.setIgnoreMouseEvents(false); // TODO this should be false by default
-
-    if (isDev) {
-      cropperWindow.openDevTools({mode: 'detach'});
-      cropperWindow.webContents.on('devtools-opened', () => {
-        setCropperWindowOnBlur();
+    if (platform === 'linux') {
+      selectorWindow = createSelector(x, y, width, height);
+      selectorWindow.on('end', () => {
+        selectorWindow = undefined;
+        mainWindow.webContents.send('cropper-window-closed');
+      });
+      selectorWindow.on('move', event => {
+        console.log(event);
+        // because GTK/GDK makes it hard to get move/resize events differently, we do this
```

Comment:
Did you see this? http://stackoverflow.com/questions/1060039/gtk-detecting-window-resize-from-the-user